Nero/src/NeroLeptons.cpp

```diff
@@ -67,18 +67,31 @@ int NeroLeptons::analyze(const edm::Event & iEvent)
         float puiso = mu.pfIsolationR04().sumPUPt;
         float totiso = chiso + TMath::Max( niso + phoiso - .5*puiso, 0. ) ;
     
-        if ( mMaxIso_mu > 0 and totiso/mu.pt() > mMaxIso_mu ) continue;
+		bool isFake = 0;
 
+        if ( mMaxIso_mu > 0 and totiso/mu.pt() > mMaxIso_mu ) continue;
+		if ( totiso/mu.pt() < 0.4){ isFake = 1; }
+		
```

Comment:
Can you confirm is not the opposite ? (totIso > XXX) the terminology fake seems misleading